### PR TITLE
Readme fixes

### PR DESCRIPTION
**Pull request by puneith**

### CODE REVIEW

**Comment by danaharon** [comment]
can you remove "The sample uses the PyAudio library to stream audio from your computer's microphone. PyAudio depends on PortAudio, which may need to be compiled when you install PyAudio." and instead add PortAudio + PyAudio to the section next to pip, gcloud?

**Comment by puneith** [comment]
@jonparrott please take a look. Need help to make sure this looks appropriate. Please, don't merge though.

File: speech/api/README.md [hunk 1]
```diff
@@ -38,25 +38,42 @@ See the
...
+
+  ```
+  $ sudo apt-get install portaudio19-dev
+  ```
 
 * If you're running the `speech_rest.py` sample:
 
     ```sh
-    $ sudo pip install -r requirements-speech_rest.txt
+    $ sudo pip install -r requirements-speech_rest.txt --ignore-installed six
```
Comment (jonparrott) [inline speech/api/README.md]: Don't use sudo, don't ignore installed six, just tell them to use virtualenv.

Comment (puneith) [inline speech/api/README.md]: I have been thinking about that as well. virtual env or conda. But, the problem is what if they want to do something outside of virtual?

Comment (puneith) [inline speech/api/README.md]: virtual / conda makes life so much easier.

Comment (jonparrott) [inline speech/api/README.md]: Then that's up to them. Virtualenv isn't a controversial recommendation. It's broadly used, and recommended by every other sample here.

**Pull request closed without merging**